[2017-04-10 16:26:56,227] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2017-04-10 16:26:56,227] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2017-04-10 16:26:56,403] [WARNING] basehttp : "GET /favicon.ico HTTP/1.1" 404 3478
[2017-04-10 16:26:56,403] [WARNING] basehttp : "GET /favicon.ico HTTP/1.1" 404 3478
[2017-04-10 16:27:19,726] [INFO] basehttp : "GET /?username=111 HTTP/1.1" 200 1471
[2017-04-10 16:27:19,726] [INFO] basehttp : "GET /?username=111 HTTP/1.1" 200 1471
[2017-04-10 16:27:29,089] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2017-04-10 16:27:29,089] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2017-04-10 16:27:29,139] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 6491
[2017-04-10 16:27:29,139] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 6491
[2018-09-03 21:15:19,839] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-03 21:15:19,839] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-03 21:15:23,563] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-03 21:15:23,563] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-03 21:17:39,228] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-03 21:17:39,228] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-03 21:17:42,171] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-03 21:17:42,171] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-04 10:19:11,160] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:19:11,160] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:19:14,703] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-04 10:19:14,703] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-04 10:35:43,065] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:35:43,065] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:35:43,828] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:35:43,828] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:35:50,530] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:35:50,530] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:35:52,565] [ERROR] exception : Internal Server Error: /login_action/
Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\mysql\base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 166, in execute
    result = self._query(query)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 322, in _query
    conn.query(q)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1057, in _read_query_result
    result.read()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1014, in _read_packet
    packet.check_error()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\python\lib\site-packages\pymysql\err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'guest_dev.auth_user' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\guest-master\sign\views.py", line 28, in login_action
    user = auth.authenticate(username = username, password = password)
  File "D:\python\lib\site-packages\django\contrib\auth\__init__.py", line 70, in authenticate
    user = _authenticate_with_backend(backend, backend_path, request, credentials)
  File "D:\python\lib\site-packages\django\contrib\auth\__init__.py", line 116, in _authenticate_with_backend
    return backend.authenticate(*args, **credentials)
  File "D:\python\lib\site-packages\django\contrib\auth\backends.py", line 16, in authenticate
    user = UserModel._default_manager.get_by_natural_key(username)
  File "D:\python\lib\site-packages\django\contrib\auth\base_user.py", line 44, in get_by_natural_key
    return self.get(**{self.model.USERNAME_FIELD: username})
  File "D:\python\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "D:\python\lib\site-packages\django\db\models\query.py", line 397, in get
    num = len(clone)
  File "D:\python\lib\site-packages\django\db\models\query.py", line 254, in __len__
    self._fetch_all()
  File "D:\python\lib\site-packages\django\db\models\query.py", line 1179, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "D:\python\lib\site-packages\django\db\models\query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "D:\python\lib\site-packages\django\db\models\sql\compiler.py", line 1066, in execute_sql
    cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 100, in execute
    return super().execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\mysql\base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 166, in execute
    result = self._query(query)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 322, in _query
    conn.query(q)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1057, in _read_query_result
    result.read()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1014, in _read_packet
    packet.check_error()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\python\lib\site-packages\pymysql\err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.ProgrammingError: (1146, "Table 'guest_dev.auth_user' doesn't exist")
[2018-09-04 10:35:52,565] [ERROR] exception : Internal Server Error: /login_action/
Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\mysql\base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 166, in execute
    result = self._query(query)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 322, in _query
    conn.query(q)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1057, in _read_query_result
    result.read()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1014, in _read_packet
    packet.check_error()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\python\lib\site-packages\pymysql\err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'guest_dev.auth_user' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\guest-master\sign\views.py", line 28, in login_action
    user = auth.authenticate(username = username, password = password)
  File "D:\python\lib\site-packages\django\contrib\auth\__init__.py", line 70, in authenticate
    user = _authenticate_with_backend(backend, backend_path, request, credentials)
  File "D:\python\lib\site-packages\django\contrib\auth\__init__.py", line 116, in _authenticate_with_backend
    return backend.authenticate(*args, **credentials)
  File "D:\python\lib\site-packages\django\contrib\auth\backends.py", line 16, in authenticate
    user = UserModel._default_manager.get_by_natural_key(username)
  File "D:\python\lib\site-packages\django\contrib\auth\base_user.py", line 44, in get_by_natural_key
    return self.get(**{self.model.USERNAME_FIELD: username})
  File "D:\python\lib\site-packages\django\db\models\manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "D:\python\lib\site-packages\django\db\models\query.py", line 397, in get
    num = len(clone)
  File "D:\python\lib\site-packages\django\db\models\query.py", line 254, in __len__
    self._fetch_all()
  File "D:\python\lib\site-packages\django\db\models\query.py", line 1179, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "D:\python\lib\site-packages\django\db\models\query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "D:\python\lib\site-packages\django\db\models\sql\compiler.py", line 1066, in execute_sql
    cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 100, in execute
    return super().execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "D:\python\lib\site-packages\django\db\backends\utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "D:\python\lib\site-packages\django\db\backends\mysql\base.py", line 71, in execute
    return self.cursor.execute(query, args)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 166, in execute
    result = self._query(query)
  File "D:\python\lib\site-packages\pymysql\cursors.py", line 322, in _query
    conn.query(q)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1057, in _read_query_result
    result.read()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 1014, in _read_packet
    packet.check_error()
  File "D:\python\lib\site-packages\pymysql\connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\python\lib\site-packages\pymysql\err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.ProgrammingError: (1146, "Table 'guest_dev.auth_user' doesn't exist")
[2018-09-04 10:35:52,756] [ERROR] basehttp : "POST /login_action/ HTTP/1.1" 500 218361
[2018-09-04 10:35:52,756] [ERROR] basehttp : "POST /login_action/ HTTP/1.1" 500 218361
[2018-09-04 10:36:55,155] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:36:55,155] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:36:57,731] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-04 10:36:57,731] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-04 10:40:39,633] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:40:39,633] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 10:40:46,548] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 10:40:46,548] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 10:40:46,568] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3120
[2018-09-04 10:40:46,568] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3120
[2018-09-04 10:41:08,678] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 302 0
[2018-09-04 10:41:08,678] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 302 0
[2018-09-04 10:41:08,694] [INFO] basehttp : "GET /accounts/login/?next=/event_manage/ HTTP/1.1" 200 1471
[2018-09-04 10:41:08,694] [INFO] basehttp : "GET /accounts/login/?next=/event_manage/ HTTP/1.1" 200 1471
[2018-09-04 10:41:08,909] [WARNING] basehttp : "GET /favicon.ico HTTP/1.1" 404 3515
[2018-09-04 10:41:08,909] [WARNING] basehttp : "GET /favicon.ico HTTP/1.1" 404 3515
[2018-09-04 10:41:18,820] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 10:41:18,820] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 10:41:18,855] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3120
[2018-09-04 10:41:18,855] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3120
[2018-09-04 18:25:49,310] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 18:25:49,310] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 18:25:54,184] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-04 18:25:54,184] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 200 1498
[2018-09-04 18:26:06,953] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 18:26:06,953] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 18:26:08,078] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 18:26:08,078] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 18:26:25,455] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 200 3529
[2018-09-04 18:26:25,455] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 200 3529
[2018-09-04 18:27:17,522] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 18:27:17,522] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 18:27:20,154] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 18:27:20,154] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 18:27:30,191] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 18:27:30,191] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 18:27:48,834] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 18:27:48,834] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 18:29:29,737] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 18:29:29,737] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 18:29:32,605] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 200 3529
[2018-09-04 18:29:32,605] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 200 3529
[2018-09-04 18:30:58,143] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 302 0
[2018-09-04 18:30:58,143] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 302 0
[2018-09-04 18:30:58,158] [INFO] basehttp : "GET /accounts/login/?next=/search_name/%3Fname%3D%25E6%2589%258B%25E6%259C%25BA HTTP/1.1" 200 1471
[2018-09-04 18:30:58,158] [INFO] basehttp : "GET /accounts/login/?next=/search_name/%3Fname%3D%25E6%2589%258B%25E6%259C%25BA HTTP/1.1" 200 1471
[2018-09-04 18:34:58,071] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 18:34:58,071] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 18:35:00,617] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 18:35:00,617] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 18:35:08,529] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 18:35:08,529] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 18:35:11,046] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 18:35:11,046] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 19:38:49,445] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 19:38:49,445] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 19:38:51,903] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 19:38:51,903] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 19:38:55,696] [INFO] basehttp : "GET /search_phone/?phone=15811112222 HTTP/1.1" 200 3343
[2018-09-04 19:38:55,696] [INFO] basehttp : "GET /search_phone/?phone=15811112222 HTTP/1.1" 200 3343
[2018-09-04 19:38:58,516] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 19:38:58,516] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 3343
[2018-09-04 19:42:22,550] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 19:42:22,550] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-04 19:42:30,168] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 19:42:30,168] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-04 19:42:31,144] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 19:42:31,144] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 3937
[2018-09-04 19:42:34,244] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 19:42:34,244] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 19:42:38,932] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 200 3529
[2018-09-04 19:42:38,932] [INFO] basehttp : "GET /search_name/?name=%E6%89%8B%E6%9C%BA HTTP/1.1" 200 3529
[2018-09-04 19:42:41,740] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 19:42:41,740] [INFO] basehttp : "GET /guest_manage/ HTTP/1.1" 200 4721
[2018-09-04 19:42:46,057] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 4721
[2018-09-04 19:42:46,057] [INFO] basehttp : "GET /search_phone/?phone= HTTP/1.1" 200 4721
[2018-09-04 19:42:52,367] [INFO] basehttp : "GET /search_phone/?phone=2222 HTTP/1.1" 200 3615
[2018-09-04 19:42:52,367] [INFO] basehttp : "GET /search_phone/?phone=2222 HTTP/1.1" 200 3615
[2018-09-04 20:20:11,748] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-04 20:20:11,748] [INFO] basehttp : "GET /search_name/?name= HTTP/1.1" 200 3937
[2018-09-05 18:01:42,818] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:01:42,818] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:01:43,010] [WARNING] basehttp : "GET /favicon.ico HTTP/1.1" 404 3515
[2018-09-05 18:01:43,010] [WARNING] basehttp : "GET /favicon.ico HTTP/1.1" 404 3515
[2018-09-05 18:01:51,378] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:01:51,378] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:01:52,302] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:01:52,302] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:02:01,385] [WARNING] basehttp : "GET /sign_index3/1/ HTTP/1.1" 404 3524
[2018-09-05 18:02:01,385] [WARNING] basehttp : "GET /sign_index3/1/ HTTP/1.1" 404 3524
[2018-09-05 18:03:37,632] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:03:37,632] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:03:45,575] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:03:45,575] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:03:46,514] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:03:46,514] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:03:49,815] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 1511
[2018-09-05 18:03:49,815] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 1511
[2018-09-05 18:03:49,919] [INFO] basehttp : "GET /static/css/draw_results.css HTTP/1.1" 200 10734
[2018-09-05 18:03:49,919] [INFO] basehttp : "GET /static/css/draw_results.css HTTP/1.1" 200 10734
[2018-09-05 18:03:49,922] [INFO] basehttp : "GET /static/js/conference_sign.js HTTP/1.1" 200 4291
[2018-09-05 18:03:49,922] [INFO] basehttp : "GET /static/js/conference_sign.js HTTP/1.1" 200 4291
[2018-09-05 18:03:49,924] [INFO] basehttp : "GET /static/js/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[2018-09-05 18:03:49,924] [INFO] basehttp : "GET /static/js/jquery-1.10.2.min.js HTTP/1.1" 200 93107
[2018-09-05 18:03:49,988] [INFO] basehttp : "GET /static/image/sign/bg-top-center.png HTTP/1.1" 200 2374
[2018-09-05 18:03:49,988] [INFO] basehttp : "GET /static/image/sign/bg-top-center.png HTTP/1.1" 200 2374
[2018-09-05 18:03:50,015] [INFO] basehttp : "GET /static/image/sign/bg-right-top.png HTTP/1.1" 200 28503
[2018-09-05 18:03:50,015] [INFO] basehttp : "GET /static/image/sign/bg-right-top.png HTTP/1.1" 200 28503
[2018-09-05 18:03:50,025] [INFO] basehttp : "GET /static/image/sign/bg-left-center.png HTTP/1.1" 200 1892
[2018-09-05 18:03:50,025] [INFO] basehttp : "GET /static/image/sign/bg-left-center.png HTTP/1.1" 200 1892
[2018-09-05 18:03:50,036] [INFO] basehttp : "GET /static/image/sign/cent.png HTTP/1.1" 200 3433
[2018-09-05 18:03:50,039] [INFO] basehttp : "GET /static/image/sign/bg-left-bottom.png HTTP/1.1" 200 57959
[2018-09-05 18:03:50,036] [INFO] basehttp : "GET /static/image/sign/cent.png HTTP/1.1" 200 3433
[2018-09-05 18:03:50,041] [INFO] basehttp : "GET /static/image/sign/bg-right-bottom.png HTTP/1.1" 200 38413
[2018-09-05 18:03:50,039] [INFO] basehttp : "GET /static/image/sign/bg-left-bottom.png HTTP/1.1" 200 57959
[2018-09-05 18:03:50,041] [INFO] basehttp : "GET /static/image/sign/bg-right-bottom.png HTTP/1.1" 200 38413
[2018-09-05 18:03:50,088] [INFO] basehttp : "GET /static/image/favicon.ico HTTP/1.1" 200 1029
[2018-09-05 18:03:50,088] [INFO] basehttp : "GET /static/image/favicon.ico HTTP/1.1" 200 1029
[2018-09-05 18:03:50,216] [INFO] basehttp : "GET /static/image/sign/mx4.ttf HTTP/1.1" 200 8173832
[2018-09-05 18:03:50,216] [INFO] basehttp : "GET /static/image/sign/mx4.ttf HTTP/1.1" 200 8173832
[2018-09-05 18:06:55,921] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 1511
[2018-09-05 18:06:55,921] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 1511
[2018-09-05 18:14:46,437] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:14:46,437] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:14:55,726] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:14:55,726] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:14:56,784] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:14:56,784] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:15:11,021] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 1511
[2018-09-05 18:15:11,021] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 1511
[2018-09-05 18:24:14,066] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:24:14,066] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:24:22,500] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:24:22,500] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:24:23,558] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4080
[2018-09-05 18:24:23,558] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4080
[2018-09-05 18:35:35,213] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:35:35,213] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:35:44,141] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:35:44,141] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:35:45,161] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:35:45,161] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:35:49,266] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 3161
[2018-09-05 18:35:49,266] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 3161
[2018-09-05 18:39:49,898] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:39:49,898] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:39:58,530] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:39:58,530] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:39:59,508] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:39:59,508] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:40:04,572] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 620
[2018-09-05 18:40:04,572] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 620
[2018-09-05 18:44:21,293] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:44:21,293] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:44:29,983] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:44:29,983] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:44:31,009] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:44:31,009] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:44:35,178] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 501
[2018-09-05 18:44:35,178] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 501
[2018-09-05 18:44:50,292] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 501
[2018-09-05 18:44:50,292] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 501
[2018-09-05 18:53:45,123] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:53:45,123] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-05 18:53:53,686] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:53:53,686] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-05 18:53:54,738] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:53:54,738] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-05 18:54:02,832] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 501
[2018-09-05 18:54:02,832] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 501
[2018-09-06 10:25:35,907] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-06 10:25:35,907] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-06 10:25:46,443] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-06 10:25:46,443] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-06 10:25:47,619] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 10:25:47,619] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 10:25:52,408] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 501
[2018-09-06 10:25:52,408] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 501
[2018-09-06 11:15:39,793] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 501
[2018-09-06 11:15:39,793] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 501
[2018-09-06 11:21:26,819] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-06 11:21:26,819] [INFO] basehttp : "GET / HTTP/1.1" 200 1471
[2018-09-06 11:21:35,407] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-06 11:21:35,407] [INFO] basehttp : "POST /login_action/ HTTP/1.1" 302 0
[2018-09-06 11:21:36,359] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:21:36,359] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:21:40,900] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 520
[2018-09-06 11:21:40,900] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 520
[2018-09-06 11:21:40,983] [WARNING] basehttp : "GET /sign_index3/1/2.png HTTP/1.1" 404 3680
[2018-09-06 11:21:40,983] [WARNING] basehttp : "GET /sign_index3/1/2.png HTTP/1.1" 404 3680
[2018-09-06 11:23:11,661] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:23:11,661] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:23:15,633] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 520
[2018-09-06 11:23:15,633] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 520
[2018-09-06 11:23:15,722] [WARNING] basehttp : "GET /sign_index3/1/2.png HTTP/1.1" 404 3680
[2018-09-06 11:23:15,722] [WARNING] basehttp : "GET /sign_index3/1/2.png HTTP/1.1" 404 3680
[2018-09-06 11:27:19,488] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:27:19,488] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:27:23,537] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 528
[2018-09-06 11:27:23,537] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 528
[2018-09-06 11:27:23,614] [WARNING] basehttp : "GET /static/2.png HTTP/1.1" 404 1639
[2018-09-06 11:27:23,614] [WARNING] basehttp : "GET /static/2.png HTTP/1.1" 404 1639
[2018-09-06 11:28:42,803] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:28:42,803] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:28:46,474] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 528
[2018-09-06 11:28:46,474] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 528
[2018-09-06 11:28:46,536] [INFO] basehttp : "GET /static/2.png HTTP/1.1" 200 711
[2018-09-06 11:28:46,536] [INFO] basehttp : "GET /static/2.png HTTP/1.1" 200 711
[2018-09-06 11:31:54,151] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:31:54,151] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:31:57,728] [ERROR] exception : Internal Server Error: /sign_index3/1/
Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\guestsign\sign\views.py", line 148, in sign_index3
    'eventNanme': event_name,'qrcode':qrcodeWithText('http://118.24.90.64:8014/sign_index/'+str(event_id))})
  File "D:\guestsign\sign\views.py", line 140, in qrcodeWithText
    img.save(savePath)
  File "D:\python\lib\site-packages\qrcode\image\pil.py", line 50, in save
    self._img.save(stream, format=format, **kwargs)
  File "D:\python\lib\site-packages\PIL\Image.py", line 1947, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: '/static/images/2.png'
[2018-09-06 11:31:57,728] [ERROR] exception : Internal Server Error: /sign_index3/1/
Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\guestsign\sign\views.py", line 148, in sign_index3
    'eventNanme': event_name,'qrcode':qrcodeWithText('http://118.24.90.64:8014/sign_index/'+str(event_id))})
  File "D:\guestsign\sign\views.py", line 140, in qrcodeWithText
    img.save(savePath)
  File "D:\python\lib\site-packages\qrcode\image\pil.py", line 50, in save
    self._img.save(stream, format=format, **kwargs)
  File "D:\python\lib\site-packages\PIL\Image.py", line 1947, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: '/static/images/2.png'
[2018-09-06 11:31:57,913] [ERROR] basehttp : "GET /sign_index3/1/ HTTP/1.1" 500 79420
[2018-09-06 11:31:57,913] [ERROR] basehttp : "GET /sign_index3/1/ HTTP/1.1" 500 79420
[2018-09-06 11:37:34,079] [ERROR] exception : Internal Server Error: /sign_index3/1/
Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\guestsign\sign\views.py", line 150, in sign_index3
    'eventNanme': event_name,'qrcode':qrcodeWithText('http://118.24.90.64:8014/sign_index/'+str(event_id))})
  File "D:\guestsign\sign\views.py", line 142, in qrcodeWithText
    img.save(savePath)
  File "D:\python\lib\site-packages\qrcode\image\pil.py", line 50, in save
    self._img.save(stream, format=format, **kwargs)
  File "D:\python\lib\site-packages\PIL\Image.py", line 1947, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: '/static/images/2.png'
[2018-09-06 11:37:34,079] [ERROR] exception : Internal Server Error: /sign_index3/1/
Traceback (most recent call last):
  File "D:\python\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "D:\python\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\guestsign\sign\views.py", line 150, in sign_index3
    'eventNanme': event_name,'qrcode':qrcodeWithText('http://118.24.90.64:8014/sign_index/'+str(event_id))})
  File "D:\guestsign\sign\views.py", line 142, in qrcodeWithText
    img.save(savePath)
  File "D:\python\lib\site-packages\qrcode\image\pil.py", line 50, in save
    self._img.save(stream, format=format, **kwargs)
  File "D:\python\lib\site-packages\PIL\Image.py", line 1947, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: '/static/images/2.png'
[2018-09-06 11:37:34,265] [ERROR] basehttp : "GET /sign_index3/1/ HTTP/1.1" 500 79359
[2018-09-06 11:37:34,265] [ERROR] basehttp : "GET /sign_index3/1/ HTTP/1.1" 500 79359
[2018-09-06 11:40:06,413] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:40:06,413] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:40:10,333] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 535
[2018-09-06 11:40:10,333] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 535
[2018-09-06 11:40:10,392] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 11:40:10,392] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 11:41:11,231] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:41:11,231] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:41:13,644] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 535
[2018-09-06 11:41:13,644] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 535
[2018-09-06 11:41:13,739] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
[2018-09-06 11:41:13,739] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
[2018-09-06 11:42:55,300] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:42:55,300] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:42:57,974] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 550
[2018-09-06 11:42:57,974] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 550
[2018-09-06 11:42:58,045] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 11:42:58,045] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 11:43:03,698] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 550
[2018-09-06 11:43:03,698] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 550
[2018-09-06 11:43:03,810] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
[2018-09-06 11:43:03,810] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
[2018-09-06 11:44:54,675] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:44:54,675] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 11:44:57,317] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 550
[2018-09-06 11:44:57,317] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 550
[2018-09-06 11:44:57,390] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 11:44:57,390] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 11:45:02,058] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 550
[2018-09-06 11:45:02,058] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 550
[2018-09-06 11:45:02,161] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
[2018-09-06 11:45:02,161] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
[2018-09-06 12:28:04,246] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 12:28:04,246] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 12:28:06,758] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 567
[2018-09-06 12:28:06,758] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 567
[2018-09-06 12:28:06,810] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 12:28:06,810] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 12:29:32,729] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 593
[2018-09-06 12:29:57,801] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 568
[2018-09-06 12:30:23,522] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 12:30:23,522] [INFO] basehttp : "GET /event_manage/ HTTP/1.1" 200 4144
[2018-09-06 12:30:26,007] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 568
[2018-09-06 12:30:26,007] [INFO] basehttp : "GET /sign_index3/1/ HTTP/1.1" 200 568
[2018-09-06 12:30:26,052] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 12:30:26,052] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 711
[2018-09-06 12:30:34,485] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 568
[2018-09-06 12:30:34,485] [INFO] basehttp : "GET /sign_index3/2/ HTTP/1.1" 200 568
[2018-09-06 12:30:34,624] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
[2018-09-06 12:30:34,624] [INFO] basehttp : "GET /static/images/2.png HTTP/1.1" 200 712
